### ISSUE TITLE
NodeTcl throws exception on Node.js 0.8 on FreeBSD

### ISSUE THREAD

**Issue by DavidChouinard** [bovine/nodetcl#6]
The following exception happens when trying to `require` NodeTcl under Node.js 0.8:

``` javascript
module.js:480
  process.dlopen(filename, module.exports);
          ^
Error: nodetcl.node: Undefined symbol "ev_default_loop_ptr"
```

The [v0.6 to v0.8 migration page](/joyent/node/wiki/API-changes-between-v0.6-and-v0.8) says that "direct use of `ev_*` and `eio_*` functions is deprecated". It seems that NodeTcl is making a call to a deprecated Node.js function.

**Comment by bovine** [comment]
Adding this define to the compilation options in wscript lets us bypass this issue by forcing /usr/local/include/node/uv-private/ev.h to declare its inlined functions as "static inline" instead of just "static":

```
-D__STDC_VERSION__=199901L
```

However, that just results in exposing joyent/node#3623 as the next blocker:

```
node example/example1.js
/libexec/ld-elf.so.1: /usr/home/jeff.lawson/opensource/nodetcl/example/nodetcl.node: Undefined symbol "_ZN2v811HandleScopeC1Ev"
*** Error code 1
```

That symbol appears to exist in the node binary:

```
$ nm /usr/local/bin/node | grep _ZN2v811HandleScopeC1Ev
000000000053f110 T _ZN2v811HandleScopeC1Ev
$ nm example/nodetcl.node | grep _ZN2v811HandleScopeC1Ev
             U _ZN2v811HandleScopeC1Ev
$ c++filt _ZN2v811HandleScopeC1Ev
v8::HandleScope::HandleScope()
```

**Comment by bovine** [comment]
Fixed in v0.8 branch by node v0.8.3-pre

The `-D__STDC_VERSION__=199901L` is not needed either.
